xml/admin_ceph_upgrade.xml

```diff
@@ -1397,6 +1397,146 @@ data:
    </step>
   </procedure>
  </sect1>
+
+ <sect1 xml:id="filestore2bluestore">
+  <title>OSD Migration to &bluestore;</title>
+  <para>
+   OSD &bluestore; is a new back end for the OSD daemons. It is the default
+   option since &productname; &productnumber;. Compared to FileStore, which
+   stores objects as files in an XFS file system, &bluestore; can deliver
+   increased performance because it stores objects directly on the underlying
+   block device. &bluestore; also enables other features, such as built-in
+   compression and EC overwrites, that are unavailable with FileStore.
+  </para>
+  <para>
+   Specifically for &bluestore;, an OSD has a 'wal' (Write Ahead Log) device
+   and a 'db' (RocksDB database) device. The RocksDB database holds the
+   metadata for a &bluestore; OSD. These two devices will reside on the same
+   device as an OSD by default, but either can be placed on faster/different
+   media.
+  </para>
+  <para>
+   In SES5, both FileStore and &bluestore; are supported and it is possible
+   for FileStore and &bluestore; OSDs to co-exist in a single cluster. During
+   the SUSE Enterprise Storage upgrade procedure, FileStore OSDs are not
+   automatically converted to &bluestore;. Be aware that the
+   &bluestore;-specific features will not be available on OSDs that have not
+   been migrated to &bluestore;.
+  </para>
+  <para>
+   Before converting to &bluestore;, the OSDs need to be running &productname;
+   &productnumber;. The conversion is a slow process as all data gets
+   re-written twice. Though the migration process can take a long time to
+   complete, there is no cluster outage and all clients can continue accessing
+   the cluster during this period. However, do expect lower performance for
+   the duration of the migration. This is caused by rebalancing and
+   backfilling of cluster data.
+  </para>
+  <para>
+   Use the following procedure to migrate FileStore OSDs to &bluestore;:
+  </para>
+  <tip>
+   <title>Turn Off Safety Measures</title>
+   <para>
+    &salt; commands needed for running the migration are blocked by safety
+    measures. In order to turn these precautions off, run the following
+    command:
+   </para>
+ <screen>
+ &prompt.smaster;salt-run disengage.safety
+ </screen>
+   <para>Rebuild the nodes before continuing:</para>
+ <screen>
+ &prompt.smaster; salt-run rebuild.node <replaceable>TARGET</replaceable>
+ </screen>
+   <para>You can also choose to rebuild each node individually. For example:</para>
+ <screen>
+   &prompt.smaster; salt-run rebuild.node data1.ceph
+ </screen>
+   <para>The rebuild is not idempotent. To address the complication of shared
```

Comment:
Two things.

1. I think the word "idempotent" needs to be explained.
2. Doesn't this contradict @swiftgist's statement in the comments?

Reply:
So, this is where I explain things but my intended audience in the conversation of this PR is the doc team.  I will go into additional details to help give enough information.  I cannot say that my explanation is perfectly suitable for the administrator reading the SES6 docs.  It's fine to cull what I said so that the manual comes from a single voice.

That said, I do somewhat expect that the technical reader of the SES6 manual would know what idempotent means.  OTOH, I do not know how many translations we support for the manual. Having a description of wouldn't hurt, but since I'm emphasizing the negative maybe a warning that `rebuild.node` will *always* remove and recreate all OSDs is more appropriate.

@lproven Where did you see the contradiction?  That will likely mean there would be cause for confusion.  The SES5 migration orchestration is idempotent.  The SES6 rebuild runner is not.